src/main/java/fyi/osm/sourdough/layers/Highways.java

```diff
@@ -80,7 +80,7 @@ public void processFeature(SourceFeature sf, FeatureCollector fc) {
         return;
       }
 
-      var isArea = sf.hasTag("area", "yes") || sf.hasTag("highway", "rest_area");
+      var isArea = sf.hasTag("area", "yes") || sf.hasTag("highway", "rest_area") || sf.hasTag("highway", "services");
```

Comment:
Minor suggestion (this is functionally the same, and I believe slightly faster)

```suggestion
      var isArea = sf.hasTag("area", "yes") || sf.hasTag("highway", "rest_area", "services");
```